hieradata_aws/class/integration/ckan_db_admin.yaml

```diff
@@ -0,0 +1,49 @@
+govuk_env_sync::tasks:
+  "pull_ckan_production_daily":
+    ensure: "present"
+    hour: "0"
+    minute: "0"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "ckan_production"
+    database_hostname: "ckan-postgresql"
+    temppath: "/tmp/ckan_production"
+    url: "govuk-production-database-backups"
+    path: "ckan-postgresql"
+  # "push_ckan_production_daily":
+  #   ensure: "present"
+  #   hour: "5"
+  #   minute: "0"
+  #   action: "push"
+  #   dbms: "postgresql"
+  #   storagebackend: "s3"
+  #   database: "ckan_production"
+  #   database_hostname: "ckan-postgresql"
+  #   temppath: "/tmp/ckan_production"
+  #   url: "govuk-staging-database-backups"
+  #   path: "ckan-postgresql"
+  "pull_ckan_pycsw_production_daily":
+    ensure: "present"
+    hour: "0"
+    minute: "0"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "ckan_pycsw_production"
```

Comment:
Unfortunately it turns out that we [currently don't push this database](https://github.com/alphagov/govuk-puppet/blob/2609a6ce355fee22457002cd33c3bc7cf4a3c80c/hieradata_aws/class/production/db_admin.yaml#L2-L13) :( so this isn't going to work out the box.

I think the simplest thing is to modify the existing db_admin boxes to start pushing and pulling this strange database and then this can go in. I'll put something together now.

Reply:
Thanks @kevindew - this should work now that #11362 is merged, right? 🤞

Reply:
Yes - well at least once that has been deployed and had a chance to dump some databases. I'm not sure how graceful/catastrophic things are when there is no database to pull.

Reply:
I've pushed up a temporary [comment-out commit](https://github.com/alphagov/govuk-puppet/pull/11361/commits/4fb2028f3dce41a30dcfdfdf47832dd98dcecfeb) to defend against this. We can revert that commit tomorrow.